llvm/lib/Transforms/Utils/SimplifyLibCalls.cpp

```diff
@@ -2797,6 +2797,21 @@ static bool insertSinCosCall(IRBuilderBase &B, Function *OrigCallee, Value *Arg,
   return true;
 }
 
+Value *LibCallSimplifier::optimizeSymmetric(CallInst *CI, bool IsEven,
+                                            IRBuilderBase &B) {
+  Value *X;
+  if (match(CI->getArgOperand(0), m_OneUse(m_FNeg(m_Value(X))))) {
```

Comment:
Please add a test for the multi-use case.

Reply:
Could you please point me to a similar test?

Reply:
Done 👍